README.md

```diff
@@ -0,0 +1,117 @@
+# NixOS on Yoga 900
+
+## Purpose
+
+This repository will provide instructions and configuration to setup your
+Yoga 900 2-in-1 laptop.
+
+It will contain manual and unassisted installation instructions separately
+so that people can understand what is happening during the unassisted
+installation script.
+
+## Prerequisites
+
+* A Yoga 900 tablet you want to install NixOS on by blowing away the Windows
+  partition.
+* A USB drive to boot NixOS boot installer image from.
+* A USB to RJ45 adapter
+* Physical network link/cable connected to Internet.
+
+## Scope
+
+* These instructions use the latest stable NixOS 16.03 minimal install image
+* We update the `nixos` root channel to a newer NixOS 16.09pre version before
+  `nixos-install`.
+* The `configuration.nix` provided will offer you a working (though minimal)
+  NixOS configuration with the following hardware verified:
+  * Wireless internet card working
+  * Webcam working
+  * Touchscreen
+  * Touchpad
+  * Reasonable power management defaults using TLP service.
+
+## Recommendations
+
+If there are BIOS or firmware updates available for your Yoga 900 available
+from Lenovo or Microsoft, please download them and install them via your
+Windows 10 partition that came with your laptop.
+
+## Manual Instructions
+
+1. Download and prepare a NixOS minimal 64-bit installer ISO from:
+   https://nixos.org/nixos/download.html under the linked with anchor
+   text _'Minimal installaction CD, 64-bit Intel/AMD'_. After download
+   make sure you verify the downloaded file's SHA-256 with the one at
+   the link under the anchor text 'SHA-256' ajacent to prior link.
+2. Write the image to a USB drive you are happy wiping contents from.
+   On Linux or OS X you can do this with the command:
+   `sudo dd bs=4M if=path/to/nixos-minimal-15.09.Y.Z-x86_64-linux.iso of=/dev/sdX`
```

Comment:
I did:
`$ curl -L https://nixos.org/releases/nixos/latest-iso-minimal-x86_64-linux >nixos-lastest.iso`
`$ sudo dd bs=4M if=nixos-latest.iso of=/dev/sdX`